packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -43,6 +43,9 @@ export const getContractTypeCategoryIcons = () =>
  * @param {array}  unsupported_list - list of unsupported contract types
  */
 
+export const showLabelForMultipliers = (checked_symbol: string) =>
+    /R_|1HZ/i.test(checked_symbol) && !/1HZ150V|1HZ250V/i.test(checked_symbol);
```

Comment:
Should check your content change for Multipliers PR as well in this case

Reply:
Just checked. You can do this with submarket as well but i like the regex better here. Easier to read

Reply:
I didn't apply regexp to the Multipliers for checking if it's a major pairs or not because you can't extract this info from symbol. Both, major and minor pairs symbols, look pretty mush the same and I need only major pairs.